app/Nova/Pirep.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\HasMany;
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Number;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\Textarea;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Laravel\Nova\Panel;
+
+class Pirep extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = 'App\Models\Smartcars\Pirep';
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'id';
+
+    public static $group = "Smartcars";
+
+    public static $with = ['bid'];
+
+    /**
+     * The columns that should be searched.
+     *
+     * @var array
+     */
+    public static $search = [];
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function fields(Request $request)
+    {
+        return [
+            ID::make()->sortable(),
+
+            BelongsTo::make('Bid', 'bid', 'App\Nova\Bid'),
+
+            Text::make('Pilot', function () {
+                return $this->account->first()->name;
+            })->onlyOnIndex(),
+
+            Textarea::make('Route', 'route')->alwaysShow(),
+
+            Number::make('Landing Rate'),
+
+            Number::make('Fuel Used'),
+
+            HasMany::make('Pilot', 'account', 'App\Nova\Account')->onlyOnDetail(),
+
+            Text::make('Flight Time'),
+
+            new Panel('Log & Comments', [
+                Textarea::make('Comments'),
+                Textarea::make('Log'),
+            ]),
+
+            new Panel('Approval Information', [
+                Boolean::make('Passed'),
+                Text::make('Pass Reason')->onlyOnDetail()->canSee(function () {
```

Comment:
These db fields are poorly named. Pass reason actually returns "Passed" or it'll return the failed reason (e.g. "you failed because you went outside of the altitude restriction at posrep X").

So this only needs to show with a failed_at